config/main.py

```diff
@@ -544,6 +546,9 @@ def reload(filename, yes, load_sysinfo):
         command = "{} -H -k {} --write-to-db".format(SONIC_CFGGEN_PATH, cfg_hwsku)
         run_command(command, display_cmd=True)
 
+    if os.path.isfile('/etc/sonic/init_cfg.json'):
+        command = "{} -j {} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename, INIT_CFG_FILE)
+
     command = "{} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename)
```

Comment:
This line should be part of an 'else' clause, Otherwise you're loading 'filename' twice if init_cfg.json exists.

Reply:
My bad and add an else statement.